Source/WebKit/UIProcess/Cocoa/WKStorageAccessAlert.mm

```diff
@@ -96,6 +123,14 @@ void displayStorageAccessAlert(WKWebView *webView, NSString *alertTitle, NSStrin
 #else
     auto alert = WebKit::createUIAlertController(alertTitle, informativeText);
 
+    if (accessoryTextList) {
```

Comment:
I don't think this null check is needed.  ObjC kind of does it for you.

Reply:
Maybe I want to check something else here. I'm passing nil from two of the functions, and I'm passing a list from the other function. I only want to create the accessory view when the list is present.